### ISSUE TITLE
Build fails - Can't open Boxes

### ISSUE THREAD

**Issue by cbinckly** [cbinckly/junos-automation-environments#1]
Both the Ubuntu and the Firefly perimeter boxes are not downlading correctly from the Vagrant/HashiCorp cloud.

Both are leaving behind the following error:
```
==> ubuntu_east: Adding box './boxes/ubuntu1404-desktop.box' (v0) for provider: virtualbox
    ubuntu_east: Downloading: ./boxes/ubuntu1404-desktop.box
An error occurred while downloading the remote file. The error
message, if any, is reproduced below. Please fix this error and try
again.
```

or 

```
    vsrx: Downloading: ./boxes/juniper-ffp-12.1X47-D15.4-packetmode.box
An error occurred while downloading the remote file. The error
message, if any, is reproduced below. Please fix this error and try
again.
```

**Comment by cbinckly** [comment]
This may be related to naming changes introduced when Vagrant moved to their new 'AppCloud'.  Box naming conventions have changed.

**Comment by cbinckly** [comment]
Fixed in 6b4f727
